edpm_ansible/ansible_plugins/module_utils/network_data_v2.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2020 OpenStack Foundation
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import os
+
+def validate_playbook_dir(playbook_dir_path):
+    if not os.path.exists(playbook_dir_path):
+        raise Exception('ERROR: Playbook directory {} does not exist.'.format(
+            playbook_dir_path))
+
+    if not os.path.isdir(playbook_dir_path):
+        raise Exception(
+            'ERROR: Playbook directory {} is not a directory'.format(
+                playbook_dir_path))
+
+
+def tags_to_dict(resource_tags):
+    tag_dict = dict()
```

Comment:
I'm not sure if this is still the case, but there was a cool blog post a few years ago about `dict()` vs `{}`. 
https://doughellmann.com/posts/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/

Be interesting to see if that is still the case in Python3.11, but I guess for this reason I have always preferenced using `variable = {}`, rather than `variable = dict()`.

Reply:
TBH, i just imported code as-is from its original module, but I think you're right about this. Let me fix it.